docs/dev/table/hive/index.md

```diff
@@ -60,7 +60,7 @@ To integrate with Hive, users need the following dependencies in their project.
 {% highlight xml %}
 <dependency>
   <groupId>org.apache.flink</groupId>
-  <artifactId>flink-connector-hive{{ site.scala_version_suffix }}</artifactId>
+  <artifactId>flink-connector-hive{{site.scala_version_suffix}}</artifactId>
```

Comment:
please revert this. It's standard to use `{{ site.scala_version_suffix }}` in Flink doc.

Reply:
Thank you very much, I will revert it.